webapp/client/src/stories/FormFieldIntegerInput.stories.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+
+import FormFieldIntegerInput from "../components/FormFieldIntegerInput";
+import StepForm from "../components/StepForm";
+import { Default as StepFormDefault } from "./StepForm.stories";
+
+export default {
+  title: "Form Fields/IntegerInput",
+  component: FormFieldIntegerInput,
+};
+
+function Template(args) {
+  return (
+    <StepForm {...StepFormDefault.args}>
+      <FormFieldIntegerInput {...args} />
+    </StepForm>
+  );
+}
+
+function TemplateLong(args) {
```

Comment:
> What do you think of the WithLongPage story?

What purpose does it serve?

Reply:
The scroll- behaviour sometimes is different if there is anything to scroll vs nothing. So I added it so one can try it out^^

Reply:
Hm interesting. Is this specific to the integer field? Or rather something general for all components?

Reply:
I only noticed it for the integer field, because of the way we handled the `type="number"` thing.

Reply:
well then that makes sense to include here!